### PR TITLE
MLPAB-534 Add page for how certificate provider wants to carry out their role

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: app/internal/page/how_would_certificate_provider_prefer_to_carry_out_their_role.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+func HowWouldCertificateProviderPreferToCarryOutTheirRole(tmpl template.Template, lpaStore LpaStore) Handler {
+	return func(appData AppData, w http.ResponseWriter, r *http.Request) error {
+		lpa, err := lpaStore.Get(r.Context(), appData.SessionID)
+		if err != nil {
+			return err
+		}
+
+		if len(lpa.PeopleToNotify) > 0 {
+			return appData.Lang.Redirect(w, r, lpa, Paths.ChoosePeopleToNotifySummary)
+		}
```
Comment (acsauk) [inline app/internal/page/how_would_certificate_provider_prefer_to_carry_out_their_role.go:27]: Why are we interested in PeopleToNotify here?

File: app/lang/en.json [hunk 2]
```diff
@@ -197,6 +197,14 @@
...
     "mobile": "Mobile number",
 
+    "howWouldCertificateProviderPreferToCarryOutTheirRole": "How would {{.FirstNames}} prefer to carry out their role?",
+    "selectOneOption": "Select one option",
+    "onlineAndByEmail": "Online and by email",
+    "onlineAndByEmailHint": "We can process your LPA application quicker if your certificate provider chooses this option.",
+    "usingPaperForms": "Using paper forms",
+    "certificateProvidersEmail": "Certificate provider's email address",
+    "certificateProviderAddress": "{{.FirstNames}}'s address",
+
```
Comment (acsauk) [inline app/lang/en.json:207]: Looks like we need `selectHowWouldCertificateProviderPreferToCarryOutTheirRole` added as a translation.

**Review by acsauk — CHANGES_REQUESTED** [review]
Had a quick play locally and looks like there's an issue when trying to progress from`HowWouldCertificateProviderPreferToCarryOutTheirRole` but not entering an email address as the conditional reveal breaks:

Screenshot 2023-01-11 at 12 37 18

And a couple of comments on the PR.

File: app/internal/page/how_would_certificate_provider_prefer_to_carry_out_their_role.go (hunk 1 above)
Comment (hawx) [reply app/internal/page/how_would_certificate_provider_prefer_to_carry_out_their_role.go:27]: ah the ol copy pasta

**Review by acsauk — APPROVED** [review]
nice nice nice :shipit:

**Pull request merged**